### ISSUE TITLE
Please Update/Extend or Abandon

### ISSUE THREAD

**Issue by karstengresch** [microprofile/microprofile-samples#52]
This repo looks extremely dated to me.

As it is still linked via https://microprofile.io/project/eclipse/microprofile-samples, it should IMHO

- either get updated (with samples extended) to MP v6.1
- or get archived/abandoned with a clear disclaimer (in this case, removing the link to this repo from https://microprofile.io/ would be wise).

Thanks!

**Comment by harimm** [comment]
Yeah. There has no activity here for the better part of a decade. Has this been abandoned?

I got here from this page - https://projects.eclipse.org/projects/technology.microprofile/developer

Tagging @agoncal
